lib/presentation/same_type_events_builder/same_type_events_list_builder.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:dartz/dartz.dart';
+import 'package:fluffychat/app_state/failure.dart';
+import 'package:fluffychat/app_state/success.dart';
+import 'package:fluffychat/presentation/same_type_events_builder/same_type_events_controller.dart';
+import 'package:fluffychat/widgets/twake_components/twake_loading/center_loading_indicator.dart';
+import 'package:flutter/material.dart';
+
+class SameTypeEventsBuilder extends StatelessWidget {
```

Comment:
please change the file name to same_type_events_builder.dart

Reply:
done